packages/cli/package.json

```diff
@@ -54,9 +54,9 @@
     "node-sass": "^4.12.0",
     "p-retry": "^4.1.0",
     "polka": "^0.5.2",
-    "preact": "^10.0.0",
-    "preact-compat": "^3.18.5",
-    "preact-render-to-string": "^5.0.6",
+    "preact": "^10.0.4",
+    "preact-compat": "^3.19.0",
```

Comment:
Is the old `compat` still needed? 🤔

Reply:
It's there just so that ppl working with Preact 8 have one less thing to do

Reply:
oh you are right! Got confused and was thinking this is the `async` package and since the new version won't work with `8.x` thought that we could remove it from there.

All good! 👍 thank you